src/convertToObject.js

```diff
@@ -3,16 +3,27 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
   // write your code here
+  const styles = {};
+
+  sourceString.split(';').map(line =>
+    line.trim()).filter(line => line)
+    .map(lns => {
```

Comment:
After using the filter method there is a new array , the purpose of the task is taking each of the items from that  array and putting them into styles object. For this purpose .forEach method can be used instead of map. Because .map method returns a new array, so usually it's used when you actually need a new array .forEach method is great for when you need to do some action with each element of the array.

Reply:
I thought since we are not supposed to use loops in these part tasks, I used .map